src/web/app/pages-instructor/instructor-home-page/instructor-home-page.component.spec.ts

```diff
@@ -28,4 +36,89 @@ describe('InstructorHomePageComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should snap with default fields', () => {
+    expect(fixture).toMatchSnapshot();
+  });
+
+  it('should snap with one course without feedback session', () => {
+    const instructorName: string = '';
+    const courseTabModels: any = {
+      course: {
+        courseId: 'CS3281',
+        courseName: 'Thematic Systems',
+        creationDate: '26 Feb 23:59 PM',
+        timeZone: 'Asia/Singapore',
+      },
+      instructorPrivilege: {
+        canModifyCourse: true,
```

Comment:
If you check the HTML for this component, you will see that based on whether these fields are true or false, the element rendered is different. A good way to add test cases would be to alter the values and see if the rendered HTML corresponds to what we want happening.